### ISSUE TITLE
Live comment output -- possible to increase expansion level (more recursion before printing [Object])

### ISSUE THREAD

**Issue by breathe** [wallabyjs/public#2108]
### Issue description or question

Hi I'm wondering -- its it possible to increase the expansion level when serializing live comment objects for display in WallabyApp?  Apologies if this is in docs somewhere and I just haven't found yet ...

```
{ orderDateTime: 'xxx',
  orderNumber: 'xxx',
  customerName: 'xxx',
  optionalOrderTime: 'xxx',
  headerItemCount: 18,
  customerPhoneNumber: 'xxx',
  optionalOrderDate: 'xxx',
  notes: [ 'xxx' ],
  paymentType: 'xxx',
  items: 
   [ { itemCount: 1,
       itemText: 'xxx',
       itemCategory: 'xxx',
       modifiers: [Object],   <--------- can I get wallabyjs to expand this?
       specialRequest: 'xxx',
       labelText: 'xxx',
       labelItemCount: 1 },
}
,... more items snipped by me
 ],
  totalItemCount: 18,
  errors: [] }
```

**Comment by smcenlly** [comment]
It's currently not possible to increase the expansion level for live comments displaying in Wallaby App. I'll mark as a feature request.

Are you familiar with Wallaby's [Value Explorer](https://wallabyjs.com/docs/intro/value-explorer.html) feature (available in VS Code and IntelliJ Editors)? It sounds like Value Explorer with [Live Comment to Auto-Expand](https://wallabyjs.com/docs/intro/value-explorer.html#live-comment-to-auto-expand) would give you what you're looking for.

**Comment by smcenlly** [comment]
We have added additional `logLimits` settings so that you can now use to increase the default expansion of logged values. Please refer to [our docs](https://wallabyjs.com/docs/config/overview.html#log-limits) for more information. The new settings are available in Wallaby core `v1.0.1286`+.
